item/views.py

```diff
@@ -0,0 +1,79 @@
+from django.contrib.auth.decorators import login_required 
+from django.db.models import Q
+from django.shortcuts import get_object_or_404, render, redirect
+
+from .forms import NewItemForm, EditItemForm 
+from .models import Item, Category
+
+def items(request):
+    query = request.GET.get('query','')
+    category_id = request.GET.get('category', 0)
+    categories = Category.objects.all()
+    items = Item.objects.filter(is_sold=False)
+
+    if category_id:
```

Comment:
if category_id is 0 its weird, better to use None not 0, does it make sense to return category_id 0 back to FE?